packages/core/src/docs/accessibility.md

```diff
@@ -59,4 +59,14 @@ const MyComponent = () => ({
 
 Colors have been designed to be accessible to as many people as possible, even those who are
 visually impaired or experiencing any kind of colorblindness. Our colors have not only been chosen
-to go well together but to also adhere to [WCAG 2.0](https://www.w3.org/TR/WCAG20/) standards.
+to go well together, but to also adhere to [WCAG 2.0](https://www.w3.org/TR/WCAG20/) standards.
+
+@### Enhanced focus indication [WCAG 2.2 focus appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)
+
+All Blueprint elements descending from an element with the `bp5-focus-enhanced-contrast` class, will
+received enhanced focus indication. With this class applied at the root, the focus contrast meets the
```

Comment:
The root part is technically inaccurate, as it doesn't have to be the root (as the first part of the sentence mentions).